src/server/worker/api_server.go

```diff
@@ -330,30 +329,37 @@ func NewAPIServer(pachClient *client.APIClient, etcdClient *etcd.Client, etcdPre
 		if err != nil {
 			return nil, fmt.Errorf("error inspecting image %s: %+v", pipelineInfo.Transform.Image, err)
 		}
-		// if image.Config.User == "" then uid, and gid don't get set which
+		if pipelineInfo.Transform.User == "" {
+			pipelineInfo.Transform.User = image.Config.User
+		}
+		if pipelineInfo.Transform.WorkingDir == "" {
+			pipelineInfo.Transform.WorkingDir = image.Config.WorkingDir
+		}
+		if server.pipelineInfo.Transform.Cmd == nil {
+			server.pipelineInfo.Transform.Cmd = image.Config.Entrypoint
+		}
+	}
+	fmt.Printf("user: %s\n", pipelineInfo.Transform.User)
```

Comment:
leftover debug statement? Or do we want this?

Reply:
Yup, good catch, removed.